packages/cel/package.json

```diff
@@ -40,11 +39,16 @@
     "./ext/strings": {
       "import": "./dist/esm/ext/strings/index.js",
       "require": "./dist/cjs/ext/strings/index.js"
+    },
+    "./internal": {
+      "import": "./dist/esm/internal.js",
+      "require": "./dist/cjs/internal.js"
```

Comment:
This will mean others will also be able to access them as well. Can't we detect the usage of node types using linter for the non test files?